.github/workflows/build-push.yml

```diff
@@ -0,0 +1,110 @@
+
+name: Build and Push Docker Image WEAC Codebase
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'python-flask/**'
+      - 'c-sharp-razor/**'
+      - 'rust-actix/**'
+      - 'ruby-sinatra/**'
+      - 'javascript-express/**'
+      - 'go-gin/**'
+      - 'go-gorilla/**'
+
+jobs:
+  build-and-push:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        service:
+          - python-flask
+          - c-sharp-razor
+          - rust-actix
+          - ruby-sinatra
+          - javascript-express
+          - go-gin
+          - go-gorilla
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: '1'
+
+      - name: Get Changed Files
+        id: changed-files
+        uses: tj-actions/changed-files@v34
+        with:
+          files: |
+            python-flask/**
+            c-sharp-razor/**
+            rust-actix/**
+            ruby-sinatra/**
+            javascript-express/**
+            go-gin/**
+            go-gorilla/**
+
+      - name: Determine if Service Needs Build
+        id: check_changes
+        run: |
+          if [[ "${{ steps.changed-files.outputs.modified_files }}" == *"${{ matrix.service }}/"* ]] || [[ "${{ steps.changed-files.outputs.added_files }}" == *"${{ matrix.service }}/"* ]]; then
```

Comment:
I tried to environment variables instead of outputs but with no luck. The issue is that soon outputs will be deprecated in workflows.

Reply:
there is no replacement for the outputs?

Reply:
But guys, are we not using `compose-prod.yml` for handling the production environment variables which need to be present at build and running time?

Reply:
There is. Environment variables. But I could not make it work.